sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -235,7 +235,7 @@ private[joins] class UnsafeHashedRelation(
       0)
 
     val pageSizeBytes = Option(SparkEnv.get).map(_.memoryManager.pageSizeBytes)
-      .getOrElse(new SparkConf().getSizeAsBytes("spark.buffer.pageSize", "16m"))
+      .get
```

Comment:
What if somehow `SparkEnv` is not initialized yet, and it returns `null`, for instance, test purpose? Looks It would throw `java.util.NoSuchElementException: None.get` after this change. Did you check all the codes paths?